R/table.R

```diff
@@ -38,3 +38,19 @@ knit_print.table = function(x, options) {
     knit_print(x, options)
   }
 }
+
+#' @export
+knit_print.summary.lm = function(x, options, ...) {
+  res = paste(c(
+    '', '', kable(x$coef, options$render.args$kable$format, ...)
+  ), collapse = '\n')
+  asis_output(res)
+}
+
+#' @export
+knit_print.lm = function(x, options, ...) {
+  res = paste(c(
+    '', '', kable(summary(x)$coef, options$render.args$kable$format, ...)
```

Comment:
I mean `lm` gives the equation of the form `dist = ? + ? speed` (instead of the R call), and `summary.lm` gives the coefficient matrix.